Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -8812,24 +8817,28 @@ def fix_security_zone_no_log_setting(
         return result
 
     @staticmethod
-    def get_all_security_rules_in_container(container: Union[Panorama, Firewall, DeviceGroup, Template, Vsys]
-                                            ) -> List[SecurityRule]:
-        """
-        Gets all the security rule objects from the given VSYS or device group and return them.
-        :param container: The object to search for the rules in, as passed to pan-os-python
-        """
+    def get_all_rules_in_container(container: Union[Panorama, Firewall, DeviceGroup, Template, Vsys],
+                                   object_class: Any):
```

Comment:
would you be able to mention what kind of objects could ```object_class``` be? 
also add a short doc-string

Reply:
Ah yes, I missed this.